happi/client.py

```diff
@@ -849,7 +849,7 @@ def from_config(cls, cfg=None):
 
         # Create our database with provided kwargs
         try:
-            database = backend(**db_kwargs)
+            database = backend(**db_kwargs, cfg_path=cfg)
```

Comment:
Reviewers note: this is not an API break since there is not yet a supported way to add and use external backends to happi, though this has been a planned feature. (There are unsupported ways to do this)